.travis.yml

```diff
@@ -14,4 +20,4 @@ before_install:
   - cd ../ruby
 script:
   - cd ../ruby && bundle exec rake
-  - cd ../python && PYTHONPATH='.' make test
+  - cd ../python && PYTHONPATH='.' sudo make test
```

Comment:
Why is the `sudo` required?

Reply:
tox creates a .tox folder and populates it with stuff (virtualenvs etc), and it needs sudo privileges to write there.

Reply:
Weird. Even to write in the source folder? Is there some other folder that one _is_ expected to write to during the build process?

Anyway, just curious. I can't think of any reason to care about it, it just seems bizarre.

Reply:
I'm not sure, but this would be a good time to mention that we're "off the travis rails".
Travis only supports one language per project in its test suites where everything is included. We have ruby and python libraries to test, so we're doing a lot more bash-ing and using it more like buildkite then a standard travis setup.